src/tokenized-line.coffee

```diff
@@ -97,25 +99,44 @@ class TokenizedLine
       leftTextLength += nextToken.value.length
       leftTokens.push nextToken
 
+    tab = leftTokens[0].buildPhantomToken(@indentLevel * @tabLength)
```

Comment:
I think we should call this `leadingWhitespace` or something to clarify it isn't technically a "tab".